Dockerfile.test

```diff
@@ -1,8 +1,10 @@
 # Set up an environment for running the tests.
-FROM fedora:29
-RUN dnf -y install 'dnf-command(builddep)'
+FROM centos:8
+RUN dnf -y install 'dnf-command(builddep)' https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
 
 # Add test dependencies
-COPY requirements.txt /requirements.txt
-RUN dnf -y install make git gettext python3-pip python3-sphinx which
-RUN pip-3 install -r requirements.txt
+COPY test-packages .
+RUN dnf -y install $(cat test-packages)
+
+# pocketlint to modify pylint behavior, and Sphinx because it isn't in EPEL8
+RUN pip-3 install pocketlint 'Sphinx == 1.7.6'
```

Comment:
Sphinx is present in the PowerTools repo, so you can enable that to install `python3-sphinx` from CentOS.

Reply:
Thanks!